sdk/hdinsight/arm-hdinsight/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/arm-hdinsight",
   "author": "Microsoft Corporation",
   "description": "HDInsightManagementClient Library with typescript type definitions for node.js and browser.",
-  "version": "0.14.0",
+  "version": "0.15.0",
```

Comment:
is there breaking change requiring major versoin bumping?

Reply:
It looks like there are only additive changes here, no breaking changes.

Reply:
thanks for confirmation, David.